### ISSUE TITLE
GATK mergemutectstats is using the id "gatk4_mergepileup"

### ISSUE THREAD

**Issue by illusional** [kids-first/kf-somatic-workflow#75]
Specifically:

https://github.com/kids-first/kf-somatic-workflow/blob/4ace465567462b61c340a75a7bab805c47c92e8c/tools/gatk_mergemutectstats.cwl#L3

Which clashes with:
https://github.com/kids-first/kf-somatic-workflow/blob/4ace465567462b61c340a75a7bab805c47c92e8c/tools/gatk_gatherpileupsummaries.cwl#L3

This can cause issues with some tools / downstream processors.

Example fix:

```diff
diff --git a/tools/gatk_mergemutectstats.cwl b/tools/gatk_mergemutectstats.cwl
index 9e0cdf3..ad715b5 100644
--- a/tools/gatk_mergemutectstats.cwl
+++ b/tools/gatk_mergemutectstats.cwl
@@ -1,6 +1,6 @@
 cwlVersion: v1.0
 class: CommandLineTool
-id: gatk4_mergepileup
+id: gatk4_mergemutectstats
 label: GATK Merge Stats
 requirements:
   - class: ShellCommandRequirement
```

**Comment by yuankunzhu** [comment]
@illusional good catch, we will fix this issue. thanks.

**Comment by illusional** [comment]
Thanks @yuankunzhu, I'll close this because it looks like the referenced issue was merged. In a PR, if you say:

> This issue fixes #75

This issue will get automatically closed when the PR merges.
